pkg/plugin/v3/webhook.go

```diff
@@ -102,6 +110,27 @@ func (p *createWebhookPlugin) Validate() error {
 			" kind and version provided", p.commandName)
 	}
 
+	// if conversion flag is set but hub and spoke version is not provided, suggest it to user and return
+	if p.conversion && (p.spoke == "" && p.hub == "") {
+		klog.V(2).Info("Hub and Spoke version can be specified in the command to scaffold Convertible and Hub interfaces")
+		return nil
+	}
+
+	// set conversion to true if hub and spoke version are specified
+	if p.hub != "" && p.spoke != "" {
+		p.conversion = true
+	}
+
+	// validate the values provided for hub and spoke
+	if p.conversion {
+		if p.spoke == "" || p.hub == "" {
+			return errors.New("Both Hub and Spoke version need to be specified when using conversion webhooks")
```

Comment:
Lets use `fmt.Errorf` not to import another library.
```suggestion
			return fmt.Errorf("Both Hub and Spoke version need to be specified when using conversion webhooks")
```